Formula/tmux.rb

```diff
@@ -44,6 +45,13 @@ class Tmux < Formula
     sha256 "b5f7bbd78f9790026bbff16fc6e3fe4070d067f58f943e156bd1a8c3c99f6a6f"
   end
 
+  # Fix occasional crash on exit.
+  # Remove with the next release (3.3).
+  patch do
+    url "https://github.com/tmux/tmux/commit/5fdea440cede1690db9a242a091df72f16e53d24.patch?full_index=1"
+    sha256 "3752098eb9ec21f4711b12d399eaa1a7dcebe9c66afc147790fba217edcf340f"
+  end
```

Comment:
Should probably be inside a `stable` block.

Reply:
Good catch!